templates/decl.j2

```diff
@@ -32,6 +32,35 @@
 
 {{ decl.doc_string | convert_markdown }}
 
+{% if decl.informal_statement %}
+
+<details open>
+    <summary>Informal translation</summary>
+
+    <div class="informal_statement">
+        <div class="statement">
+            {{ decl.informal_statement | convert_markdown }}
+        </div>
```

Comment:
I'm worried the mathjax rendering is going to have quite a large impact on page load times. Could we maybe:

* Defer mathjax rendering of `.informal_statement` until the `details` is expanded
* Put a "expand informal math" setting in the sidebar next to the color scheme setting, so that users can opt to collapse everything by default and get the associated performance boost.

Reply:
I think the `details` should be collapsed by default, not for performance but because the translations are misleadingly sketchy:
> Right now I put it in a `<details>` menu, expanded by default to make browsing this demo easier. I think they should be collapsed by default in production, at least until the translations improve.

But a toggle also sounds like a good idea.

Do you know how to defer the rendering?

Reply:
There's a (possibly outdated) question about deferring here: https://stackoverflow.com/a/27952213/102441

Reply:
[This page in the documentation](https://docs.mathjax.org/en/latest/web/typeset.html#handling-asynchronous-typesetting) also seems relevant.

Enabling [Lazy Typesetting](https://docs.mathjax.org/en/latest/output/lazy.html#lazy-typesetting-1) would also be a reasonable option.